nav2_dwb_controller/dwb_plugins/test/twist_gen.cpp

```diff
@@ -222,6 +230,16 @@ TEST(VelocityIterator, dwa_gen)
   checkLimits(twists, 0.0, 0.125, -0.1, 0.1, -0.16, 0.16, hypot(0.125, 0.1), 0.0, 0.1);
 }
 
+TEST(VelocityIterator, dwa_gen_zero_frequency)
+{
+  auto nh = makeTestNode("dwa_gen");
+  nh->declare_parameter("controller_frequency", 0.0);
+  LimitedAccelGeneratorTest gen;
+  gen.initialize(nh, "dwb");
+  // Default value should be 0.05
+  EXPECT_EQ(gen.getAccelerationTime(), 0.05);
```

Comment:
Test both ways, so now change the parameter and initalize again to make sure its what it should be with 1 / controller_frequency.

Reply:
This is to make sure that we're testing that this code is doing what we think it is, not only calling to get some default values. It can help us find issues where different inputs give us unexpected outputs due to branching